{"frames":[{"func":"unique(data2)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in unique(data2) : objeto 'data2' no encontrado\n"}